test/mithril.route.js

```diff
@@ -130,14 +139,42 @@ describe("m.route()", function () {
 		expect(m.route()).to.equal("/a")
 	})
 
+	dit("skips route change if component ctrl.onunload calls preventDefault when the route has parameters", function (root) { // eslint-disable-line
```

Comment:
Nit: ditch the space between `function` and the opening parenthesis here and throughout the test.